app/assets/stylesheets/base/_variables.scss

```diff
@@ -0,0 +1,43 @@
+// Typography
+$base-font-family: $helvetica;
+$heading-font-family: $base-font-family;
+
+// Font Sizes
+$base-font-size: 1em;
+$h1-font-size: $base-font-size * 2.2;
+$h2-font-size: $base-font-size * 1.6;
+$h3-font-size: $base-font-size * 1.3;
+$h4-font-size: $base-font-size * 1.25;
+$h5-font-size: $base-font-size * 1.15;
+$h6-font-size: $base-font-size;
+
+// Line height
+$base-line-height: 1.5;
+$heading-line-height: 1.2;
+
+// Other Sizes
+$base-border-radius: 3px;
+$base-spacing: $base-line-height * 3.5em;
+$secondary-spacing: $base-line-height * 1.5em;
+$small-spacing: $base-spacing / 2;
+$base-z-index: 0;
+
+// Colors
+$teal: #3CC9AE;
+$pink: #FF7875;
+$dark-pink: #F95652;
+$dark-gray: #434343;
+$white: #FFF;
+
+// Font Colors
+$base-background-color: $teal;
+$base-font-color: $dark-gray;
+$action-color: $pink;
+
+// Border
+$base-border-color: $white;
+$base-border: 1px solid $base-border-color;
+
+// Forms
+$form-box-shadow: inset 0 1px 3px rgba(#000, 0.06);
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.